code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -74,11 +74,9 @@
 		. += "<span class='notice'>The baton is [round(cell.percent())]% charged.</span>"
 	else
 		. += "<span class='warning'>The baton does not have a power source installed.</span>"
+	. += "<span class='notice'>When turned on this item will knockdown anyone it hits after a short delay. While on harm intent, this item will also do some brute damage, even if turned on.</span>"
```

Comment:
```suggestion
	. += "<span class='notice'>When turned on, this item will knockdown anyone it hits after a short delay. While on harm intent, this item will also do some brute damage, even if turned on.<br>\
	This item can be recharged in a recharger. Using a screwdriver on this item will allow you to access its power cell, which can be replaced.</span>"
```
	


"weapons charger" is called "recharger" in game, might want to just call it as is or a new player might not find it